### PR TITLE
Auto-initialize SDK

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**

### CODE REVIEW

File: Example/Info.plist [hunk 1]
```diff
@@ -35,5 +35,11 @@
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 	</array>
+	<key>RASApplicationIdentifier</key>
+	<string>RAS_APPLICATION_IDENTIFIER</string>
+	<key>RASProjectSubscriptionKey</key>
+	<string>RAS_SUBSCRIPTION_KEY</string>
+	<key>RMAConfigAPIEndpoint</key>
+	<string>https://miniapp-platform-japaneast-dev.azurewebsites.net</string>
```
[the remark this answers is not in the document]
Comment (ts-leojoseph-r) [reply Example/Info.plist]: Updated with "https://www.example.com". Using any other link could mislead. Thanks.

File: .travis.yml [hunk 2]
```diff
@@ -30,4 +30,6 @@ deploy:
     branch: master
 
 notifications:
-  slack: rakuten:9bxA8UeOrx5XOIWggBcEewPV
\ No newline at end of file
+  slack:
+    rooms:
+      secure: yx82MU+0Q/+JrqHTTa01MN+nhkrLELq7tzHDzbhgrI2z2XxRYXX1asaz/5EGFAZOoOBR3DzqHJTvWOC2I5rjZDHBfht0ecSir1Ml8ThHAjzavjI/kJpP0SeHRgUNcz9KVmlX5dKAYBezbSoXs9sIEReigLpajt3skT10tmGMkMOUwJCcxS29NfuVE8x3bOKKKG9XrygM8chA7OV6nYm2tF42OkiJyBghYI36/fpAlpK09sRzybuFFAeRJIWEzD0vQnZp3gw9D590iZzcvFThgtsWsyzE1RiVYnnjYopHeoP9cTwHo1qbsTsWvdecBsrBK3DIwnGGEsy4ehArfMyqf36jxNpbNXdsanknBHtdq4O7+Bn2lCgZEfwh/RUhmCFtRxV07V4YmuKPYyFEwqx7WFp6nVerkV3sns6ZPW70MEC93DyFHJ09APHPnLLFyIvlo9WhE8mUO2NKCx15EXplFt0Svd2cuw1VlwC4aWpiy0jhTNnx0ci+mVn/Nw07jZQiI5aphqA4wRoij5HtHfXVRAMmAm0h1qtfGkwziVEvw3ju8Zf0AFckK6BC+yt2MNFddFXB0V/ZjuSW43XEirALFEZdCbInY+JYFjPBlM0J7l0BnDkFHypEjvUAexvWk58WO8wXQXyNaF3VI9gdLxXRKo5DhF/l20RRuyP/a3hqkDc=
```
[the remark this answers is not in the document]
Comment (ts-leojoseph-r) [reply .travis.yml]: Yes, I have added them now.

File: Example/Info.plist (hunk 1 above)
[the remark this answers is not in the document]
Comment (donnie-jp) [reply Example/Info.plist]: The URL can be injected at build time by using https://docs.fastlane.tools/actions/set_info_plist_value/ or similar to inject a secret env variable.

[the remark this answers is not in the document]
Comment (ts-leojoseph-r) [reply Example/Info.plist]: Thank you. I will update the same.

**Comment by donnie-jp** [comment]
The changes look ok but I don't see any code that does auto-initialization.

**Comment by ts-leojoseph-r** [comment]
> The changes look ok but I don't see any code that does auto-initialization.

I think the ticket was meant to setup the configurations for SDK in the plist and to be used/retrieved by the SDK.
https://jira.rakuten-it.com/jira/browse/MINI-181

And the Hosting app/ Sample app is going to access the public exposed APIs using MiniApp.<interface>.
Am I mission something here :thinking_face:

**Comment by donnie-jp** [comment]
> > The changes look ok but I don't see any code that does auto-initialization.
> 
> I think the ticket was meant to setup the configurations for SDK in the plist and to be used/retrieved by the SDK.
> https://jira.rakuten-it.com/jira/browse/MINI-181
> 
> And the Hosting app/ Sample app is going to access the public exposed APIs using MiniApp..
> Am I mission something here :thinking_face:

That's fine then. The title made me think it was supposed to include something like https://github.com/rakutentech/ios-remote-config/blob/master/RRemoteConfig/LoaderObjC.m

**Pull request merged**